src/codegen/generators/pou_generator.rs

```diff
@@ -689,6 +689,11 @@ impl<'ink, 'cg> PouGenerator<'ink, 'cg> {
         // cannot use index from members because return and temp variables may not be considered for index in build_struct_gep
         let mut var_count = 0;
         for m in members.iter().filter(|it| !it.is_var_external()) {
+            if m.is_property() {
+                // We want to ignore properties
+                continue;
+            }
```

Comment:
self-note: Another approach might be removing the internally created property variable after lowering method calls. However, the following code will then no longer return a duplicate symbol error, which is _should_ be incorrect given properties are accessed like variables (e.g. `instance.myprop := 5;`).
```
VAR
  foo : DINT
END_VAR

PROPERTY foo : DINT
  // ...
END_PROPERTY
```